sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -1260,3 +1260,61 @@ test_suite(
         redact.lower(),
     ) for lc_state in structs.to_dict(CONST.LCV) for redact in REDACT],
 )
+
+[opentitan_multislot_flash_binary(
+    name = "boot_policy_valid_img_a_{}_b_{}".format(
```

Comment:
Sorry for missing this earlier but can you choose a name that doesn't conflict with other targets in this file? E.g. `sigverify_always_a_{}_b_{}`

Reply:
This is intentional because this rule is generating the binaries `boot_policy_valid_img_a_bad_b_empty` and `boot_policy_valid_img_a_empty_b_bad`.  Therefore the test is using the `boot_policy_valid_img_a_bad_b_bad` previously generated by the `boot_policy_valid` test. 
Is that a problem? If so I would problably need one `opentitan_multislot_flash_binary` for each of the cases.

Reply:
Thanks for the explanation. Not a problem, I thought this could cause conflicts but seems like we're ok and I'll do another pass in this file anyway. As a side note: since we bootstrap unconditionally and building flash images is pretty cheap we don't really need to worry about sharing flash images between tests .